docs/DeveloperGuide.md

```diff
@@ -41,54 +41,54 @@ license, making it open source and available for anyone to use and modify.
     - [Create a new tag](#create-a-new-tag)
     - [Tag a `Food`](#tag-a-food)
     - [Untag a `Food`](#untag-a-food)
-    - [List `Food` by tag(s).](#list-foods-by-tags)
-    - [Delete `Food` by tag(s).](#delete-foods-by-tags)
+    - [List `Food` by tag(s).](#list-food-by-tags)
+    - [Delete `Food` by tag(s).](#delete-food-by-tags)
```

Comment:
Is it okay to remove the fullstop at the end of the description? Same for View Food in table of content.

Reply:
Resolved!